src/Card.tsx

```diff
@@ -159,4 +165,37 @@ export default class Card {
             }
         }
     }
+
+    static fromLine(line: string, type: CardType = CardType.Default): Card {
+        if (FORMAT_MTGO.test(line)) {
+            let m = FORMAT_MTGO.exec(line);
+            if (m && m.groups) {
+                let card = new Card(m.groups.card, Number(m.groups.quantity), type);
+                card.collectorNumber = m.groups.number;
+                card.setCode = m.groups.set;
+
+                return card;
+            }
+        }
+
+        if (FORMAT_MTGA.test(line)) {
+            let m = FORMAT_MTGA.exec(line);
+            if (m && m.groups) {
+                return new Card(m.groups.card, Number(m.groups.quantity), type);
+            }
+        }
```

Comment:
Could you add a comment about this being a regex-group (I wasn't familiar with this at first, and thought: Hey where is this groups object coming from?)